kubernetes/helm/pinot/values.yaml

```diff
@@ -117,6 +115,14 @@ controller:
 
   podAnnotations: {}
 
+  # For example, in private GKE cluster, you might add cloud.google.com/load-balancer-type: Internal
+  serviceAnnotations:
+    type: "service"
```

Comment:
`controller.service.annotations` already service that purpose. Also `type: "service"` and should not be present by default.